### ISSUE TITLE
API 19 Crash

### ISSUE THREAD

**Issue by Hackmodford** [wseemann/FFmpegMediaMetadataRetriever#102]
When I tried using the library on API 19 I get this error.

```
06-18 13:46:51.807 6785-6785/com.hackmodford.bigfinish E/AndroidRuntime: FATAL EXCEPTION: main
                                                                         Process: com.hackmodford.bigfinish, PID: 6785
                                                                         java.lang.UnsatisfiedLinkError: dlopen failed: cannot locate symbol "stpcpy" referenced by "libffmpeg_mediametadataretriever_jni.so"...
                                                                             at java.lang.Runtime.loadLibrary(Runtime.java:364)
                                                                             at java.lang.System.loadLibrary(System.java:526)
                                                                             at wseemann.media.FFmpegMediaMetadataRetriever.<clinit>(FFmpegMediaMetadataRetriever.java:124)
                                                                             at com.hackmodford.bigfinish.content.ContentBuilder.<init>(ContentBuilder.java:20)
                                                                             at com.hackmodford.bigfinish.content.ContentBuilder.getInstance(ContentBuilder.java:32)
                                                                             at com.hackmodford.bigfinish.database.DBHandler.getAllAlbums(DBHandler.java:266)
                                                                             at com.hackmodford.bigfinish.database.DBHandler.<init>(DBHandler.java:99)
                                                                             at com.hackmodford.bigfinish.database.DBHandler.getInstance(DBHandler.java:87)
                                                                             at com.hackmodford.bigfinish.BigFinish.onCreate(BigFinish.java:40)
                                                                             at com.android.tools.fd.runtime.BootstrapApplication.onCreate(BootstrapApplication.java:369)
                                                                             at android.app.Instrumentation.callApplicationOnCreate(Instrumentation.java:1007)
                                                                             at android.app.ActivityThread.handleBindApplication(ActivityThread.java:4344)
                                                                             at android.app.ActivityThread.access$1500(ActivityThread.java:135)
                                                                             at android.app.ActivityThread$H.handleMessage(ActivityThread.java:1256)
                                                                             at android.os.Handler.dispatchMessage(Handler.java:102)
                                                                             at android.os.Looper.loop(Looper.java:136)
                                                                             at android.app.ActivityThread.main(ActivityThread.java:5017)
                                                                             at java.lang.reflect.Method.invokeNative(Native Method)
                                                                             at java.lang.reflect.Method.invoke(Method.java:515)
                                                                             at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run(ZygoteInit.java:779)
                                                                             at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:595)
                                                                             at dalvik.system.NativeStart.main(Native Method)```

**Comment by wseemann** [comment]
Are you using the prebuilt AAR's or the gradle module?

**Comment by Hackmodford** [comment]
Gradle, latest .10 version.
I will be testing API 21 shortly...

**Comment by Hackmodford** [comment]
Any idea how to fix this?

**Comment by xperiafan13-rom** [comment]
@Hackmodford  How to fix this?

**Comment by wseemann** [comment]
Use the prebuilt AAR's.

**Comment by xperiafan13-rom** [comment]
IM ALREADY USE THIS, BUT SAME ERROR

**Comment by wseemann** [comment]
Why are you responding in caps?

**Comment by xperiafan13-rom** [comment]
I'm sorry, I was writing text in capital letters and left so.

**Comment by Hackmodford** [comment]
I will test using the prebuilt AAR's

**Comment by wseemann** [comment]
Can you test this AAR and let me know if it solves your issues? I modified the build scripts to use a dynamic APP_PLATFORM based on the architecture being built.

[all-fmmr.aar.zip](https://github.com/wseemann/FFmpegMediaMetadataRetriever/files/349166/all-fmmr.aar.zip)

**Comment by Hackmodford** [comment]
The AAR above fixed the problem.

**Comment by wseemann** [comment]
Fixed in the latest release, version 1.0.11.
